website/docs/r/contract_subject_one_way_filter.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Contract"
+layout: "aci"
+page_title: "ACI: aci_contract_subject_one_way_filter"
+sidebar_current: "docs-aci-contract-subject-one-way-filter"
+description: |-
+  Manages ACI Filter
+---
+
+# aci_filter #
+
+Manages ACI Filter
+
+NOTE: The aci_contract_subject_one_way_filter should only be used with datasources as it will create conflicts with the aci_contract_subject resource
+
+## API Information ##
+
+* `Class` - vzRsFiltAtt
+* `Distinguished Name` - uni/tn-{tenant_name}/brc-{contract_name}/subj-{contract_subject_name}/intmnl/rsfiltAtt-{tnVzFilterName}
+
+## GUI Information ##
+
+* `Location` - Tenant -> Contract -> Contract Subject -> Contract Subject One Way Filter
```

Comment:
Tenants -> {tenant_name} -> Contracts -> Standard -> {contract_name} -> {subject_name} -> Filter Chain For Consumer to Provider / Filter Chain For Provider to Consumer